addons/acre/XEH_postInit.sqf

```diff
@@ -0,0 +1,5 @@
+#include "script_component.hpp"
+
+if (EGVAR(common,acre)) then {
+    call FUNC(init);
```

Comment:
Please add an setting to disable whole ACRE component functionalities, I don't want to have it forced as it can eventually collide with some missions setups.

Reply:
Done.